word2vec_decade_experiment/01_word2vec_decade_runner.py

```diff
@@ -0,0 +1,115 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:biovectors]
+#     language: python
+#     name: conda-env-biovectors-py
+# ---
+
+# # Run Word2vec on abstracts for each Decade
```

Comment:
Just trying to get more of the big picture idea here. So you're training your word embedding model here and then using the model to embed gene and diseases and looking at how similar different representations of gene, disease pairs are?

Reply:
>So you're training your word embedding model here and then using the model to embed gene and diseases and looking at how similar different representations of gene, disease pairs are?

Exactly. We are using this model to develop latent vectors for genes and diseases based on how they appear in biomedical abstracts. The key here is the model is unbiased as it knows nothing about biology or anything and we are still able to recover decent number of associations.